src/features/Instructors/data/slice.js

```diff
@@ -90,6 +82,16 @@ export const instructorsSlice = createSlice({
     deleteRowSelect: (state, { payload }) => {
       state.rowsSelected = state.rowsSelected.filter(row => row !== payload);
     },
+    addInstructorRequest: (state) => {
+      state.addInstructor.status = RequestStatus.LOADING;
+    },
+    addInstructorSuccess: (state, { payload }) => {
+      state.addInstructor.status = RequestStatus.SUCCESS;
+      state.addInstructor.data = payload;
+    },
+    addInstructorFailed: (state) => {
+      state.addInstructor.status = RequestStatus.ERROR;
+    },
   },
```

Comment:
I'm not sure about this names, there aren't clear, maybe:

```suggestion
    setInstructorLoading: (state) => {
      state.addInstructor.status = RequestStatus.LOADING;
    },
    setInstructorSuccessRequest: (state, { payload }) => {
      state.addInstructor.status = RequestStatus.SUCCESS;
      state.addInstructor.data = payload;
    },
    setInstructorErrorRequest: (state) => {
      state.addInstructor.status = RequestStatus.ERROR;
    },
  },
```

Reply:
Usually when we defined a reducer is not a good practice write the name as a setter, I will find a name more descriptive

Reply:
I do not modify this, for me it is quite clear to describe the actions that the reducer performs and adding the request word does not contribute to this description